ouroboros/ouroboros/management/commands/load_test_data.py

```diff
@@ -0,0 +1,42 @@
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+import factory
+
+from django_seed import Seed
+# why is this called 'hacker'?
+from hacker import models
+
+class Command(BaseCommand):
+    def handle(self, **options):
+        load_test_data()
+
+class HackerFactory(factory.Factory):
```

Comment:
Is this factory used? It doesn't seem to be, but maybe it is used implicitly somehow